plutus-core/untyped-plutus-core/Language/UntypedPlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -121,6 +123,15 @@ data CekUserError
     | CekEvaluationFailure -- ^ Error has been called or a builtin application has failed
     deriving (Show, Eq)
 
+-- FIXME: should this be the same errorcodes as the typed-plutus-core.CekUserError original datatype?
+instance ErrorCode Language.UntypedPlutusCore.Evaluation.Machine.Cek.CekUserError where
+      errorCode
+        Language.UntypedPlutusCore.Evaluation.Machine.Cek.CekEvaluationFailure {}
```

Comment:
Ditto.

Reply:
This, and the previous is TH-generated haskell code, hence the ugly fully-qualified names :see_no_evil: